karma.conf.js

```diff
@@ -0,0 +1,73 @@
+// Karma configuration
+// Generated on Wed Apr 12 2017 02:56:16 GMT+0700 (ICT)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['jasmine'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
+      'node_modules/angular/angular.js',
+      'node_modules/angular-mocks/angular-mocks.js',
+      'node_modules/angular-route/angular-route.js',
+      'public/**/*.js',
+      'test/**/*.spec.js'
```

Comment:
you will need to include a mock stub module before including the test files here.
as you can see, the test fails with `MELTER` not found. this  constant is injected in bootstrap.coffee.
so what i would do is to include a file something like
`'test/js/meltingpot.js',`
and add these code inside it
```
angular.module("meltingpot")
.constant('MELTER', {})
.value('greeter', {
  getGreeting: function(name) { return "Hello " + name; }
})
```
this will initialized all of the injectors required by meltingpot angular app